.goreleaser.yml

```diff
@@ -80,11 +80,11 @@ dockers:
       - "golangci/golangci-lint:v{{ .Major }}.{{ .Minor }}-alpine"
 
 brews:
-  - github:
+  - tap:
```

Comment:
LGTM.

Need to update a couple of repos personally as well :)

Reply:
yeah, I just did it on my repos